mcf-browser/src/PageBar.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {Component} from 'react';
+
+ interface PageBarPropType {
+     /** Tracks which page the user is currently viewing (0-indexed) */
+     page: number;
+
+     /** Stores the maximum number of pages */
+     maxPage: number;
+
+     /** The function to go to the next page */
+     goToNextPage: Function
+
+     /** The function to go to the previous page */
+     goToPrevPage: Function;
+
+     /** The function to go to a certain page */
+     goToPage: Function;
+ }
+
+/** Component for rendering the pagination bar */
+class PageBar extends Component<PageBarPropType> {
+  /** Constructor for the component
+        * @param {PageBarPropType} props the props passed in
+       */
+  constructor(props: PageBarPropType) {
```

Comment:
if there is no state, the constructor actually isn't needed!

Reply:
done!